src/fitnesse/testsystems/slim/SlimTestContextImpl.java

```diff
@@ -14,6 +15,15 @@
   private final Map<String, String> symbols = new HashMap<String, String>();
   private final Map<String, ScenarioTable> scenarios = new HashMap<String, ScenarioTable>();
   private final TestSummary testSummary = new TestSummary();
+  private TestPage pageToTest;
+
+  public SlimTestContextImpl(TestPage pageToTest) {
+    this.pageToTest = pageToTest;
+  }
+  
+  public SlimTestContextImpl() {
+    super();
+  }
```

Comment:
No, it's no longer needed. I tried to remove, but then I got a few compilation errors on unit tests that instantiated this class to test something else. Since TestPage would serve as optional "context" data I figured to just leave the default constructor and unit tests as it is.

Want me to pull request again removing it? I'll propably have to mock a TestPage instance for unit tests that are instantiating SlimTestContextImpl.